### PR TITLE
fix: TOOLS-3053 validate role names for manage acl command

### PR DESCRIPTION
**Pull request by a-spiker**
This PR introduces robust validation for role names and role existence throughout the Aerospike Admin tool, ensuring that only valid and existing roles can be used in user and role management commands. It also adds comprehensive unit tests to cover a wide range of valid and invalid scenarios.

- Centralized all role validation in util.validate_roles.
   - Role names must now be:
   - Validation checks both the format and existence of roles.
   - Clear, context-aware error messages are logged for invalid roles or names.
- All relevant manage commands (user creation, role assignment, role creation, etc.) now use the centralized validation.
   - The list of valid roles is dynamically fetched from the cluster before validation.
- Added and expanded unit tests in test/unit/utils/test_util.py & test_manage_controller:
   - Valid role names (including edge cases)
   - Roles not present in the valid roles list
   - Logging/error message assertions

### CODE REVIEW

File: lib/live_cluster/manage_controller.py [hunk 1]
```diff
@@ -269,6 +269,11 @@ async def _do_default(self, line):
 
         if self.warn and not self.prompt_challenge():
             return
+        
+        resp = await self.cluster.admin_query_roles(nodes="principal")
+        valid_roles = list(resp.values())[0].keys()
+        if not util.validate_roles(roles, valid_roles):
```
[the remark this answers is not in the document]
Comment (a-spiker) [reply lib/live_cluster/manage_controller.py:275]: not required.. cachiing might cause complications of expiry and management etc
